themes/default/content/docs/intro/concepts/config.md

```diff
@@ -240,3 +240,11 @@ Console.WriteLine($"Active: {data.GetProperty("active")}");
 {{% /choosable %}}
 
 {{< /chooser >}}
+
+## Special Configuration Options
+
+This is a list of configuration keys that the Pulumi CLI is aware of:
+
+| Tag                                 | Type             | Description                                                                                               |
+|-------------------------------------|------------------|-----------------------------------------------------------------------------------------------------------|
+| `pulumi:disable-default-providers` | Sequence[String] | Packages for which default providers should be disabled. `*` disables default providers for all packages. |
```

Comment:
This doesn't quite make clear what a `Sequence[String]` is or how it should be used. Would it maybe make sense to adjust this to include an example here as well, like those in the Resources > Providers doc?

Reply:
Addressed in https://github.com/pulumi/pulumi-hugo/pull/1050/commits/0a31b126113168ca5148d5a20c0bcef3089752a8